### DISCUSSION TITLE
Does vue-apollo V4 supports SSR?

### DISCUSSION THREAD

**Discussion by ElvisUpUp** [Akryum/vue-apollo#1474] [Q&A]
Does vue-apollo V4 supports SSR? The V4 document about SSR is out-dated, and I wonder if V4 supports SSR or not. If it support now, I appreciate to contribute the document about the part of SSR. Please let me know, thank you. @Akryum

**Comment by Olli1080** [comment]
Yes it does support SSR. Had to have some tries before i got it right. Have a look at this for a reference 
https://v4.apollo.vuejs.org/guide-advanced/ssr.html#create-apollo-client or
https://github.com/Olli1080/FS-Homepage/blob/custom_cms/frontend/src/shared/graphql.ts
